packages/ui-signer/src/Extrinsic.tsx

```diff
@@ -3,32 +3,52 @@
 // of the ISC license. See the LICENSE file for details.
 
 import { I18nProps } from '@polkadot/ui-app/types';
-import { QueueTx } from './types';
+import { RxFees } from '@polkadot/api-observable/types';
+import { Fees, QueueTx } from './types';
 
 import BN from 'bn.js';
 import React from 'react';
 import { Trans } from 'react-i18next';
-import { Method } from '@polkadot/types';
+import { Balance, Method } from '@polkadot/types';
 import { Call, Modal } from '@polkadot/ui-app/index';
 import IdentityIcon from '@polkadot/ui-react/IdentityIcon';
+import withMulti from '@polkadot/ui-react-rx/with/multi';
+import withObservable from '@polkadot/ui-react-rx/with/observable';
 
+import FeeDisplay from './Fees';
 import translate from './translate';
 
 type Props = I18nProps & {
   children?: React.ReactNode,
-  value: QueueTx
+  onChangeFees: (hasAvailable: boolean) => void,
+  value: QueueTx,
+  fees: RxFees
 };
 
 class Transaction extends React.PureComponent<Props> {
   render () {
-    const { children, t, value: { accountId, accountNonce = new BN(0), extrinsic } } = this.props;
+    const { children, fees, style, t, value: { accountId, accountNonce, extrinsic } } = this.props;
+
+    const nonce = accountNonce
+      ? accountNonce
+      : new BN(0);
 
     if (!extrinsic) {
       return null;
     }
 
+    const methodInstance = new Method(extrinsic, extrinsic.meta);
+    const amount = methodInstance.args[1].raw;
+    const recipientId = methodInstance.args[0].raw.toString();
```

Comment:
We should never access `.raw` outside of types. As it stands Amuary is working on removing all `.raw`

Reply:
I discussed this with Amaury, which resulted in the following PR to update user-facing docs https://github.com/polkadot-js/api/pull/375, and I'll use that technique instead of using `.raw`